web-app/packages/lib/src/modules/project/views/ProjectSettingsViewTemplate.vue

```diff
@@ -146,7 +165,7 @@ export default defineComponent({
       this.settings.access.writersnames = newSettings.access.writersnames
       this.saveSettings(newSettings)
     },
-    saveSettings: debounce(function (newSettings) {
```

Comment:
This was grouping few edits in the same API request, no?

Reply:
This is only cleanup of debounce. It is still running without debouncing.

Reply:
Yeah, restore debounce